src/main/resources/swagger.api/schemas/innReachTransactionSortingParametersDTO.json

```diff
@@ -0,0 +1,80 @@
+{
+  "description": "INN-Reach Transaction sorting parameters",
+  "type": "object",
+  "properties": {
+    "types" : {
+      "description": "List of INN-Reach Transaction types",
+      "type": "array",
+      "items": {
+        "type": "string",
+        "enum" : [
```

Comment:
common enum types can be defined in the separate json schema and then reused in many places. in this case the same enum can be reused in innReachTransactionsDTO.json and innReachTransactionSortingParametersDTO.json.

see the following file as an example of enum definition: https://github.com/folio-org/mod-kb-ebsco-java/blob/master/ramls/types/packages/contentTypeEnum.json